docs/samples/Microsoft.ML.Samples/Dynamic/SDCACalibrated.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class SDCACalibrated_BinaryClassificationExample
```

Comment:
>SDCACalibrated_BinaryClassificationExample [](start = 17, length = 42)

if you sync we master you will find what we now have Trainers folder with Binary/Regression/etc.
Can you put your sample where?
Also please don't use `_` in your class names and methods :)

Reply:
Ok.

---
In reply to: [256210290](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256210290) [](ancestors = 256210290)